pkg/svcat/service-catalog/plan.go

```diff
@@ -155,3 +186,57 @@ func (sdk *SDK) RetrievePlanByClassAndPlanNames(className, planName string,
 	}
 	return &searchResults.Items[0], nil
 }
+
+// CreatePlan creates a new plan.
+func (sdk *SDK) CreatePlan(opts CreatePlanOptions,
+) (Plan, error) {
+	if opts.Scope == AllScope {
+		return nil, errors.New("invalid scope: all")
+	}
+	fromPlan, err := sdk.RetrievePlanByName(opts.Name, ScopeOptions{
```

Comment:
Oops, sorry I need to remove the approval, this is a bug.

It should be retrieving the existing plan using `opts.From` instead of the new plan name.

Here's what you will see if you run it from the command line, to see the bug in action:

```console
$ svcat get plans 
# view the listing of existing plans to pick an existing plan name for the command below

$ svcat create plan --from existingplan newplan
Error: plan not found 'newplan'
```